todo_app/tests/test_app.py

```diff
@@ -10,244 +12,27 @@ def client():
-                "cover": {
-                    "idAttachment": None,
-                    "color": None,
-                    "idUploadedBackground": None,
-                    "size": "normal",
-                    "brightness": "light",
-                    "idPlugin": None
-                }
-            },
-            {
-                "id": "603e2d668ecdb44ae6add243",
-                "checkItemStates": None,
-                "closed": False,
-                "dateLastActivity": "2021-03-02T12:19:51.004Z",
-                "desc": "",
-                "descData": None,
-                "dueReminder": None,
-                "idBoard": "abcd1234",
-                "idList": "600ede7caeb0c67b818ec913",
-                "idMembersVoted": [],
-                "idShort": 31,
-                "idAttachmentCover": None,
-                "idLabels": [],
-                "manualCoverAttachment": False,
-                "name": "moo",
-                "pos": 81920,
-                "shortLink": "TvCH1dTC",
-                "isTemplate": False,
-                "cardRole": None,
-                "badges": {
-                    "attachmentsByType": {
-                        "trello": {
-                            "board": 0,
-                            "card": 0
-                        }
-                    },
-                    "location": False,
-                    "votes": 0,
-                    "viewingMemberVoted": False,
-                    "subscribed": False,
-                    "fogbugz": "",
-                    "checkItems": 0,
-                    "checkItemsChecked": 0,
-                    "checkItemsEarliestDue": None,
-                    "comments": 0,
-                    "attachments": 0,
-                    "description": False,
-                    "due": "2021-04-01T00:00:00.000Z",
-                    "dueComplete": False,
-                    "start": None
-                },
-                "dueComplete": False,
-                "due": "2021-04-01T00:00:00.000Z",
-                "idChecklists": [],
-                "idMembers": [],
-                "labels": [],
-                "shortUrl": "https://trello.com/c/TvCH1dTC",
-                "start": None,
-                "subscribed": False,
-                "url": "https://trello.com/c/TvCH1dTC/31-moo",
-                "cover": {
-                    "idAttachment": None,
-                    "color": None,
-                    "idUploadedBackground": None,
-                    "size": "normal",
-                    "brightness": "light",
-                    "idPlugin": None
-                }
-            }
-        ]
-        response.json.return_value = json_return
-        return response
-    return None
\ No newline at end of file
+    assert b'To Do List' in response.data
+    assert b'Items in Progress' in response.data
+    assert b'Completed Items' in response.data
+
+@mongomock.patch(servers=(('fakemongo.com', 27017),))
+def test_add_item():
```

Comment:
This test seems to be verifying that mongomock is working, rather than our app, is that right? That's not necessarily a problem, but might be worth explicitly catching in the test's name